client/src/components/SearchForm.js

```diff
@@ -118,6 +129,26 @@ class SearchForm extends React.Component {
     }));
   }
 
+  buildQuery(data, filterEmpty = true) {
+    // If the data is already a string, return it as-is
```

Comment:
code style opinion--these lines are sort of self explanatory, so the comments aren't really needed. (Comments in general are best for "gotcha" type situations, or where it's a really odd line of code, and you have no idea why it's there at face value.)

Reply:
I may or may not have copy pasted that from Stack Overflow :P

Removed the comments 👍